index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+	<title>News Summary App</title>
+	<link rel="stylesheet" href="./styles/styles.css">
+</head>
+<body>
+		<h1>News Summary App</h1>
+		<section id='news-slot'>
+		</section>
+		<div class='container'>
+			<div class="row">
+				<div class="col-md news-block">
```

Comment:
Good use of divs within divs. This really shows in the good layout of your site. Clear and easy to follow with good classNames